_maps/RandomRuins/IceRuins/icemoon_underground_hermit.dmm

```diff
@@ -69,6 +69,11 @@
 /obj/machinery/hydroponics/soil,
 /turf/open/floor/grass/fairy,
 /area/ruin/powered/hermit)
+"vy" = (
+/obj/item/cultivator/rake,
+/obj/item/cultivator,
```

Comment:
Not recommending anything but if a rake is a subtype of a cultivator it might have all of the abilities of the cultivator making the cultivator unnecessary clutter but that's just a theory and who says hermits are meant to be clean and organized.

Reply:
Yeah i was just listening to people here. https://github.com/tgstation/tgstation/pull/82219. They want rakes i say give them rakes!